tutorials/howto/howto_new_profile_version.md

```diff
@@ -0,0 +1,177 @@
+* [Specifications repository](https://github.com/BioSchemas/specifications): to add compliant examples regarding the new draft profile, you need editing rights
+
+If you do not have the right permissions to edit the mentioned resources, send an email to the [mailing list](mailto:public-bioschemas@w3.org) explaining why you should have access and what Bioschemas group you belong to. Once your case has been validated, you will get an invitation to join the GDrive and the repositories.
+
+From now on, we will use ScholarlyArticle as an example to create and publish a new draft profile.
+
+## 2. Create a new profile crosswalk
+
+A profile crosswalk is the cornerstone of Bioschemas profile development. It is a spreadsheet with the Schema.org fields to be considered together with cardinality, marginality, suggested controlled vocabularies and examples. It is not a regular spreadsheet but one with a template and some predifined functionalities. It is then translated into strcutured data so that the Bioschemas web site is automatically updated. 
+
+A new profile is commonly based on the previous one so the easiest way to start is from the previous version.
+
+* Go to the [GDrive](https://drive.google.com/drive/folders/0B8yXU9SkT3ftaWJtTGYyTTJjck0) and navigate to the folder [Specifications](https://drive.google.com/drive/folders/0Bw_p-HKWUjHoNThZOWNKbGhOODg)
+* Locate and open the folder corresponding to the profile you want to update, in our example it is the [ScholarlyArticle](https://drive.google.com/drive/folders/1CgEyta4d7vFfYT7r7w9HtLZ4kAidDmJI) folder
+* Create a copy of the latest ScholarlyArticle Mapping file. At the time of writing, it was the ScholarlyArticle Mapping 0.2-DRAFT
+* Rename the copy so it reflects the new draft version, in this case it would be ScholarlyArticle Mapping 0.3-DRAFT
+
+### 2.1. The Specification Info tab
+
+In this tab, you need to update the "Description" column. In most cases the actual description of the profile will remain the same but the "Summary of Changes" will change
+* List all the changes in this new draft profile
+
+You will also need to update: 
+* the "version" column with the new version
+* the "Full Example" column with the link pointing to examples for this new version, see more on the [Update examples](#6-update-examples) section
+
+You should update the column "Official Type" if and only if you modified the main parent type from Schema.org. For instance, in previous versions the Bioschemas TrainingMaterial profile had CreativeWork as the main parent Schema.org type but it was changed to LearningResource from version 0.8
+
+### 2.2. The Schema.org mapping tab
+
+Here you will make the changes to the profile. There are a couple of possibilities here.
+* You want to update a property that already exists (e.g., there is typo, description should be updated, a new controlled vocabulary will be added, a new example is necessary, expected Schema.org types have changed, etc.)
```

Comment:
I think it would be easier to read if this top level of nesting was made a level 4 heading.

Reply:
I am not sure I understand this comment

Reply:
I found this section hard to read. I was wondering if a structure more like the following would help.

```md
#### Update Existing Property
You want to update a property that already exists (e.g., there is typo, description should be updated, a new controlled vocabulary will be added, a new example is necessary, expected Schema.org types have changed, etc.)
* Go to the column you want to change and update it as needed

#### Add a New Property
You want to add a new property
  * If it is a property belonging to a parent type already recorded, see Figure 1 below. Our ScholarlyArticle example already extends ScholarlyArticle, Article, CreativeWork and Thing
    * Add a new column under the corresponding type. Let's suppose you want to add the ...
```